ratelimit/redis.go

```diff
@@ -213,37 +218,49 @@ func (c *clusterLimitRedis) Delta(clearText string) time.Duration {
 	return d
 }
 
+func setError(span opentracing.Span, b bool) {
+	if span != nil {
+		ext.Error.Set(span, b)
+	}
+}
```

Comment:
I think this should only set `true` so we need to drop `b bool` and call it only where it is needed

Reply:
true